net-libs/telepathy-qt/telepathy-qt-0.9.8.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+inherit python-single-r1 cmake-utils virtualx
```

Comment:
python deps are apparently only added for BDEPEND and DEPEND (test), why the switch to python-single-r1?